src/app/shared/menu/date/custom-date-range-dropdown.component.spec.ts

```diff
@@ -118,9 +118,12 @@ describe('CustomDateRangeDropdownComponent', () => {
     expect(comp.googleAnalyticsEvent).toHaveBeenCalledWith(comp.CUSTOM_DATE, comp.tempRange);
   });
 
-  it('should not close the dropdown on window scroll', () => {
+  it('should not close the dropdown on window scroll', done => {
     comp.open = true;
+    window.addEventListener('scroll', (e) => {
```

Comment:
it honestly makes a bit more sense that we'd need some kind of asynchronous test on these actually